dashboard/test/lib/pd/jot_form/form_questions_test.rb

```diff
@@ -11,18 +11,21 @@ class FormQuestionsTest < ActiveSupport::TestCase
         @questions = [
           TextQuestion.new(
             id: 1,
+            order: 1,
```

Comment:
I added `order` to each question here both to more accurately represent our real data (JotForm Qs always have an order), and also because for some reason the sort order changed without it and failed tests. I presume [sort_by](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/lib/pd/jot_form/form_questions.rb#L69) with all `nil` values has indeterminate results.

Reply:
Thats perfect